Index.html

```diff
@@ -214,40 +214,6 @@ <h2 class="hh2">Multi-Post Story</h2>
           </div>
 
       </article>
-      <article class="contact-section" id="contact-section">
-        <section class="cc-sect cc-top">
-            <p class="contact mble">CONTACT US</p>
-            <p class="text-invite">
-                If you have an application you are interested in developing, 
-                a feature that you need built or a project that needs coding. 
-                I’d love to help with it 
-            </p>
-        </section>
-
-        <section class="cc-sect">
-            <form action="https://formspree.io/f/mgeqvqbb" method="post" class="myform" id="someform">
-                <input type="text" maxlength="30" name="Name" required placeholder="Full Name">
-                <input type="email" name="Email" required placeholder="Email" id="email">
-                <textarea maxlength="500" name="message" required placeholder="Enter You Message Here"></textarea>
-                <button type="submit" class="mmb">See Project</button>
-            </form>
-            <img src="image/contact-bar.png" alt="horizontal bar" class="contact-image">
-        </section>
```

Comment:
- [ ] You Implemented all the project requirement but for when you commit last commit You removed all the requirement So Kindly Add again and submit for review.